examples/hello-world-parallel/hello_world_parallel.bal

```diff
@@ -14,4 +14,7 @@ public function main() {
     worker w3 {
         io:println("Hello, World! #k");
     }
+
+    // Wait for all workers to finish.
+    _ = wait {w1, w2, w3};
```

Comment:
why do we need this? AFAIR we anyway wait for all the workers to finish

Reply:
It does not wait, within the test invocations. It does wait if we run it through a ballerina run only.

Reply:
The main function is invoked from tests then we can't guarantee that all workers in the main function have finished. To ensure that they are finished added a wait so that it would wait for all the workers to complete

Reply:
So what I'm wondering is whether the fix would be to simply wait for whatever workers started to be finished in test runs? isn't that the case?

Reply:
Shall we fix this in test case itself, without introducing wait functionality? 

A better solution is, add multiple attempts with a fixed time interval to validate the output.

Reply:
@hasithaa as discussed added the proper fix with 88c1c7c